style.css

```diff
@@ -576,3 +569,437 @@ footer {
 }
 
 /* -----------ENDS SECTION CONTACTME MOBILE------------- */
+
+@media (max-width: 767px) {
+  .navlink:first-child,
+  .navlink:nth-child(2),
+  .navlink:nth-child(3),
+  .navlink:nth-child(4),
+  .heythere,
+  .envelope,
+  .se,
+  .desktopgheading2,
+  .desktopgheading3,
+  .container-cards .card:nth-child(2) .card-heading:nth-child(2),
+  .container-cards .card:nth-child(3) .card-heading:nth-child(2),
+  .container-cards .card:nth-child(5) .card-heading:nth-child(2),
+  .container-cards .card:nth-child(6) .card-heading:nth-child(2),
+  .image-hover {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .hamburger-image,
+  .image-hover {
+    display: none;
+  }
+
+  .navlink {
+    margin-left: 4vh;
+    font-size: 15px;
+    line-height: 20px;
+    height: 7vh;
+  }
+
+  .button-list {
+    display: flex;
+    justify-content: column;
+    height: 7vh;
+    align-items: center;
+  }
+
+  .navlink:first-child {
+    margin-left: 62vw;
+  }
+
+  .navlink:nth-child(4) {
+    margin-right: 7vw;
+  }
+
+  .container-landingpage {
+    display: grid;
+    grid-template-columns: 6% 90%;
+    grid-template-rows: 45% 13% auto;
+    background-image: url(img/image-header-desktop.png);
+    background-size: 34%;
+    padding-bottom: 18vh;
+  }
+
+  .container-logos {
+    grid-area: 1/1/2/2;
+    flex-direction: column;
+    padding-left: 3vh;
+    width: 6vw;
+    height: 31vh;
+    margin-top: 17vh;
+  }
+
+  .container-experience {
+    grid-area: 3/2/4/3;
+    width: 50%;
+  }
+
+  .hey {
+    font-size: 72px;
+    line-height: 81.5px;
+  }
+
+  .paragraph {
+    font-size: 20px;
+    line-height: 32px;
+  }
```

Comment:
Great going so far 🔥. We have some tiny `font` requirements to meet here 🤝 

- [ ] Kindly make sure you stick to the Figma template requirements as closely as possible. The `font family` for the Desktop version is `Inter`. Please make sure you check all the other differences in fonts and fix them. ♻️ 

![image](https://github.com/eamador1/portfolio-project/assets/117144028/ec45d746-e83a-4310-ab37-cd10a611c555)